README.md

```diff
@@ -2,4 +2,53 @@
 
 **THIS IS DRAFT AF. DON'T READ IT IF YOU DON'T LIKE CONFUSION.**
 
-## TOC
\ No newline at end of file
+## TOC
+
+* [A "Shit" Example](#a-shit-example)
+
+## A "Shit" Example
+
+My kids are allowed tasteful explitives.
+"Shit"—for example—is fine in my house.
+My mom—however—hates the word.
+
+I tell Rock—my 7 year old—"there's nothing wrong with the word 'shit'.
+Mr. Roger's loved the word.
+But you're grandma hates it.
+So don't say it around her.
+Use 'poop' instead."
+
+"Who is Mr. Rogers?", he'll asks.
+I think he gets it.
+
+Let's implement this scenareo of "shit", Rock, and grandma's house using React's Context API (>= v16.4).
+
+```jsx
+// It's ok to say "shit" as a default.
+let ExpletiveContext = React.createContext("shit");
+
+// Some people don't like the word "shit".
+// Use a differnt word, when in their company.
+// It's good manners.
+let ContextualExclamation = () => (
+  <ExpletiveContext.Consumer>
+    {word => <span>Oh {word}!</span>}
+  </ExpletiveContext.Consumer>
+);
+
+// Grandma *hates* the word "shit".
+// Say "poop" at Grandma's house.
+let GrandmasHouse = props => (
+  <ExpletiveContext.Provider value="poop" {...props} />
```

Comment:
Most people don't realize `{...props}` includes children until they've specifically thought about this so they might not get how nesting works.

Reply:
It's also not very clear from this example how this is different from just passing props. Maybe a component in between would help.

Reply:
I agree. Do you have examples in the docs or elsewhere that do a good job of illustrating deeply nested components without being contrived? I have a contrived example in the "create, consume, provide" section.

Would it be fair to link to a CodeSanbox with a more "real-world" example?

Reply:
This isn't a bad one I think. You can see there's a thing in between

<img width="502" alt="screen shot 2018-04-03 at 12 40 15 am" src="https://user-images.githubusercontent.com/810438/38221480-9f4df412-36d7-11e8-888d-a6d4f919c6f1.png">

Reply:
Cool. I plan to continue with the "bare-bones" example, with a follow-up example that introduces the intermediate components.